console/console-init/ui/src/modules/msg-and-iot/dailogs/components/ProjectTypeConfiguration/ProjectTypeConfiguration.tsx

```diff
@@ -2,13 +2,22 @@ import React from "react";
 import { Radio, Title } from "@patternfly/react-core";
 
 interface IProjectTypeConfigurationProps {
-  selectedStep?: string;
-  setSelectedStep: (value: string) => void;
+  selectedStep?: string | "messaging" | "iot";
+  setSelectedStep: (value: "messaging" | "iot") => void;
 }
 const ProjectTypeConfiguration: React.FunctionComponent<IProjectTypeConfigurationProps> = ({
   selectedStep,
   setSelectedStep
 }) => {
+  const onChange = (_: any, event: any) => {
```

Comment:
why two events are arguments here?

Reply:
This the required definition of onChange for radio component- `onChange?: (checked: boolean, event: React.FormEvent<HTMLInputElement>) => void; ` since checked is unused variable so, i replaced it with `_` to remove warning.

Reply:
In that case can we update the type as boolean?